<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogBoxViewPosition</class>
 <widget class="QDialog" name="DialogBoxViewPosition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View Position</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>eye X</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxDirectionX">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99.989999999999995</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxDirectionY">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99.989999999999995</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>40</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>direction  X</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxEyeY">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99.989999999999995</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxDirectionZ">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99.989999999999995</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxEyeZ">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99.989999999999995</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonViewFrom">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>eye Z</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>direction Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>direction Z</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>eye Y</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxEyeX">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>69</width>
     <height>26</height>
    </size>
   </property>
   <property name="minimum">
    <double>-99.989999999999995</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
